core/startos/src/net/service_interface.rs

```diff
@@ -5,25 +5,13 @@ use serde::{Deserialize, Serialize};
 use ts_rs::TS;
 
 use crate::net::host::binding::BindOptions;
-use crate::net::host::HostKind;
-use crate::prelude::*;
 
 #[derive(Clone, Debug, Deserialize, Serialize, TS)]
 #[ts(export)]
 #[serde(rename_all = "camelCase")]
-pub struct ServiceInterfaceWithHostInfo {
+pub struct ServiceInterfaceWrapper {
     #[serde(flatten)]
     pub service_interface: ServiceInterface,
-    pub host_info: ExportedHostInfo,
-}
-
-#[derive(Clone, Debug, Deserialize, Serialize, TS)]
-#[ts(export)]
-#[serde(rename_all = "camelCase")]
-pub struct ExportedHostInfo {
-    pub id: HostId,
-    pub kind: HostKind,
-    pub hostnames: Vec<ExportedHostnameInfo>,
 }
```

Comment:
you should leave this type as is - just use `ServiceInterface` instead of it in patch-db